WcaOnRails/config/cable.yml

```diff
@@ -6,4 +6,5 @@ test:
 
 production:
   adapter: redis
-  url: redis://localhost:6379/1
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  channel_prefix: dummy_production
```

Comment:
I guess it should be `wca_production`, but this config doesn't seem relevant anyway.